### PR TITLE
add nil expr for summarise and mutate - Py Polars `lit(None)`

### PR DESCRIPTION
**Pull request by lkarthee**
This PR add `lit(None)`.  Can be used to align `describe` with Py Polars. 

```elixir
df = DF.new(strs: ["a", "b", "c"], nums: [1, 2, 3])
DF.mutate(df, c: nil)
#Explorer.DataFrame<
  Polars[3 x 3]
  strs string ["a", "b", "c"]
  nums s64 [1, 2, 3]
  c null [nil, nil, nil]
>
DF.summarise(df, c: nil)
#Explorer.DataFrame<
  Polars[1 x 1]
  c null [nil]
>
```

I will add test cases after review.

### CODE REVIEW

**Review by josevalim — APPROVED** [review]

File: test/explorer/data_frame_test.exs [hunk 1]
```diff
@@ -663,6 +663,16 @@ defmodule Explorer.DataFrameTest do
...
 
   describe "mutate/2" do
+    test "with nil" do
+      df = DF.new(strs: ["a", "b", "c"], nums: [1, 2, 3])
+      assert DF.mutate(df, c: nil) |> inspect() == ~s(#Explorer.DataFrame<
+  Polars[3 x 3]
+  strs string ["a", "b", "c"]
+  nums s64 [1, 2, 3]
+  c null [nil, nil, nil]
+>)
```
Comment (josevalim) [inline test/explorer/data_frame_test.exs:673]: We should not rely on inspect for tests. We should use something like to_columns or assert on individual series. The tests below are a good example!

Comment (lkarthee) [reply test/explorer/data_frame_test.exs]: what is preferred in `DF.dtypes(df1)` or `df1.dtypes`

Comment (josevalim) [reply test/explorer/data_frame_test.exs]: Any is fine. See the one we use the most in test and follow that template :)

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

File: test/explorer/data_frame_test.exs (hunk 1 above)
Comment (lkarthee) [reply test/explorer/data_frame_test.exs]: i have rewritten tests and pushed..

**Pull request merged**